TRACONBoundaries.geojson

```diff
@@ -492,17 +492,53 @@
+    {"type":"Feature","properties":{"id":"ZLXN","prefix":["ZLXN"],"name":"Xining Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[102.025,35.778055555556],[101.83361111111,35.776944444444],[101.82305555556,35.779027777778],[101.74133333333,35.795555555556],[101.65497222222,35.819111111111],[101.57141666667,35.848611111111],[101.49127777778,35.883833333333],[101.41516666667,35.924555555556],[101.34363888889,35.970444444444],[101.27725,36.021194444444],[101.21647222222,36.076361111111],[101.16183333333,36.135611111111],[101.11366666667,36.198444444444],[101.07244444444,36.264416666667],[101.03841666667,36.333027777778],[101.01188888889,36.40375],[100.99308333333,36.476055555556],[100.98216666667,36.549416666667],[100.97925,36.623277777778],[100.98436111111,36.697055555556],[100.9975,36.770166666667],[101.01855555556,36.842111111111],[101.04744444444,36.912277777778],[101.08388888889,36.980194444444],[101.12772222222,37.04525],[101.17855555556,37.107027777778],[101.23602777778,37.164972222222],[101.29969444444,37.218694444444],[101.36911111111,37.26775],[101.44372222222,37.31175],[101.52294444444,37.350361111111],[101.57577777778,37.374138888889],[101.58333333333,37.375277777778],[102.04111111111,37.438333333333],[103.01638888889,37.520833333333],[102.87,36.520555555556],[102.76527777778,35.782222222222]]]]}},
```

Comment:
I'm not familiar with this airspace, but there is a significant overlap of airspace here. Can you confirm that this is supposed to be this way? @RudiZhang 
![Screen Shot 2022-09-19 at 12 30 21](https://user-images.githubusercontent.com/967239/190998872-df667c10-ad0e-41f0-8ce2-6da9d7447859.png)

Reply:
Yep, depends on the altitude.